src/Control/Carrier/Parse/Simple.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE GADTs, GeneralizedNewtypeDeriving, TypeOperators, UndecidableInstances #-}
+-- | A carrier for 'Parse' effects suitable for use in the repl, tests, etc.
+module Control.Carrier.Parse.Simple
```

Comment:
By contrast, I wrote this module by starting with the other one and then paring it down significantly.